### ISSUE TITLE
Dependency issue - anvil_extras

### ISSUE THREAD

**Issue by meatballs** [anvilistas/amoni#10]
### Discussed in https://github.com/anvilistas/amoni/discussions/9

<div type='discussions-op-text'>

<sup>Originally posted by **psychedel** August 24, 2022</sup>
Thank you for the awesome tool @owen.campbell !
May I ask - how to add anvil_extras as working dependency?

I receive error
```

The app you requested could not be loaded.

This app may be misconfigured. The following error occurred:

App dependency not found

```

But I did everything from manual and anvil_extras is the single dependency of my app..
I've completed this step -
```
amoni app add --as-dependency https://github.com/anvilistas/anvil-extras.git anvil_extras C6ZZPAPN4YYF5NVJ
```

I see in my config.yaml that
```
dep_id:
  C6ZZPAPN4YYF5NVJ: anvil_extras
```
And I have anvil_extras in my app folder.
And in anvil.yaml of my Anvil application I see the same dependency id
```
- app_id: C6ZZPAPN4YYF5NVJ
  version: {dev: false}
```</div>

**Comment by meatballs** [comment]
I'm seeing the same behaviour. Not sure why yet.

**Comment by meatballs** [comment]
Got it! It's a typo in `config.yaml`

`dep_id` should be `dep-id`

If you change that manually, it should get things working. I'll push a fix for that when I get a moment.

**Comment by meatballs** [comment]
Fixed in 0.0.9
